### DISCUSSION TITLE
为什么1.18.2的optifine的某些版本会与推荐版本相冲突

### DISCUSSION THREAD

**Discussion by eewwewewefs** [Meloong-Git/PCL#2905] [提问] [answered]
在下载了装有optifine和其相应的frroge推荐版本时，在无装载任何模组的情况下启动游戏，pcl2会弹出窗口说此optifine与下载的forge版本相冲突，目前不确定是不是我的电脑的自身缘故

**Comment by burningtnt** [comment, marked as answer]
请查阅 https://optifine.net/downloads 以获取 OptiFine 与 Forge 版本的兼容性

**Comment by hejiehao** [comment]
发一下具体的 OptiFine 和 Forge 版本

**Comment by eewwewewefs** [comment]
主要出现冲突的版本为optifineH9和forge40.2.9,我不确定是不是电脑原因

**Comment by hejiehao** [comment]
复现失败
你可能需要发送启动器日志

**Comment by kong-que-dnf123** [comment]
~话说forge40.2.9在列表里显示不是最老的可选forge吗（）~
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/a0feaf1c-781f-4d4c-a5cd-47ced589ddcf)

**Comment by FQO3** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/111474347/76e11bb2-95bb-4600-b099-7a6774893744)
同样复现失败，能提供一下日志文件吗？

**Comment by WForst-Breeze** [comment]
超时关闭，如有信息可以继续补充😉
